services/distribution/src/main/java/app/coronawarn/server/services/distribution/dgc/client/ProdDigitalCovidCertificateClient.java

```diff
@@ -67,6 +67,13 @@ public ValueSet getValueSet(String hash) throws FetchValueSetsException {
         FetchBusinessRulesException::new);
   }
 
+  @Override
+  public List<BusinessRuleItem> getBnRules() throws FetchBusinessRulesException {
+    return getResponseAndTreatExceptions(digitalCovidCertificateClient::getBnRules,
+        "business rules",
```

Comment:
the string argument here is used to log exceptions about failed calls. There are also another method using "business rules" entity name. Maybe would not be bad to specifically say here "booster" in order to differentiate the loggers between the two.